rangeproof/qrgroup.go

```diff
@@ -0,0 +1,83 @@
+package rangeproof
+
+import (
+	"github.com/privacybydesign/gabi/big"
+	"github.com/privacybydesign/gabi/keyproof"
+)
+
+// QrGroup represents the group of quadratic residues modulo n = p*q, i.e. ((Z/nZ)*)^2
+// where p, q, (p-1)/2 and (q-1)/2 are all prime.
+type (
+	QrGroup struct {
+		N    *big.Int // RSA modulus
+		R, S *big.Int // Base points in QR_n
+	}
```

Comment:
Yes this is a significant ammount of code duplication, however either suggested fix is not great in my opinion as for
a) G, H are really not reasonable to use in this context, they are specific bases to the revocation proof and not even generated for issuers for which revocation is not enabled, so using them here is improper.
b) adding those as base points either leaves G, H unset because of similar reasons as above, which is also improper.

Long term, the best solution here is to merge these files into some sort of common base library, however, the exact details of what this library should have need some carefull consideration and further engineering, which in my opinion put this outside of the scope of the agreement made for this work with VWS, which made me go for this solution out of economic considerations. I am willing to put hours into fixing this in a better way if there is agreement from VWS to spend that time.